content/hardware/02.hero/boards/uno-rev3/features.md

```diff
@@ -6,14 +6,6 @@ Arduino UNO is a microcontroller board based on the **ATmega328P**. It has 14 di
 
 <FeatureList>
 
-<Feature title="ATmega328P" image="core">
-
-The classic high-performance, low-power AVR® microcontroller. 
-
-<FeatureLink title="Datasheet" url="https://content.arduino.cc/assets/Atmel-7810-Automotive-Microcontrollers-ATmega328P_Datasheet.pdf" download blank/>
-
-</Feature>
-
 <Feature title="Replaceable chip" image="configurability">
 
 The ATmega328P can easily be replaced, as it is not soldered to the board.  
```

Comment:
Can you add a link to EEPROM?

https://docs.arduino.cc/learn/programming/eeprom-guide